client/package.json

```diff
@@ -20,7 +20,6 @@
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
     "@vitejs/plugin-react": "^4.3.3",
-    "env-cmd": "^10.1.0",
```

Comment:
I see we use `env-cmd` in the `preview` command

Reply:
It's moved to the root package.json

Reply:
or use workspaces

Reply:
Reverted back, will think about workspaces later

Reply:
I think we should keep the samples simple and avoid workspaces.
Ideally a customer would want to just use one of packages on its own as a starting point